### PR TITLE
[Enhancement] Add Chromium SCAP STIG content

### PR DESCRIPTION
**Pull request by redhatrises**
- Adds Chromium SCAP content for the "chromium-browser" package that is included
  in the RH Supplementary channels for RHEL6

### CODE REVIEW

File: Chromium/transforms/shorthand2xccdf.xslt [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+
+  <xsl:template match="fileowner-desc-macro">
+    To properly set the owner of <xhtml:code><xsl:value-of select="@file"/></xhtml:code>, run the command:
+    <xhtml:pre xml:space="preserve">$ sudo chown <xsl:value-of select="@owner"/><xsl:text> </xsl:text><xsl:value-of select="@file"/> </xhtml:pre>
+  </xsl:template>
+
+  <xsl:template match="filegroupowner-desc-macro">
+    To properly set the group owner of <xhtml:code><xsl:value-of select="@file"/></xhtml:code>, run the command:
+    <xhtml:pre xml:space="preserve">$ sudo chgrp <xsl:value-of select="@group"/><xsl:text> </xsl:text><xsl:value-of select="@file"/> </xhtml:pre>
+  </xsl:template>
```
Comment (shawndwells) [inline Chromium/transforms/shorthand2xccdf.xslt]: unused macros should be cleaned out (sysctl, filegroupowner, etc)

**Comment by redhatrises** [comment]
@shawndwells cleaned up the xslt macros.

**Comment by iankko** [comment]
@redhatrises 

1) ```Chromium/Makefile``` should be updated to reflect the recent required ```$(PROD)``` argument:
```
# git diff Makefile
diff --git a/Chromium/Makefile b/Chromium/Makefile
index d769528..65acc14 100644
--- a/Chromium/Makefile
+++ b/Chromium/Makefile
@@ -38,7 +38,7 @@ shorthand2xccdf: shorthand-guide
 
 checks:
        xmlwf $(IN)/checks/*.xml
-       $(SHARED)/$(TRANS)/combinechecks.py $(CONF) $(IN)/checks > $(OUT)/unlinked-$(PROD)-oval.xml
+       $(SHARED)/$(TRANS)/combinechecks.py $(CONF) $(PROD) $(IN)/checks > $(OUT)/unlinked-$(PROD)-oval.xml
        xmllint --format --output $(OUT)/unlinked-$(PROD)-oval.xml $(OUT)/unlinked-$(PROD)-oval.xml
 
 guide: shorthand2xccdf
```
Without this change ```make``` in ```Chromium``` folder returns following error:
```
./shared/transforms/combinechecks.py ../config input/checks > output/unlinked-chromium-oval.xml
make: *** [checks] Error 1
[root@localhost Chromium]# ../shared/transforms/combinechecks.py ../config input/checks
Provide a directory name, which contains the checks.
....
```

2) Chromium content should be added into the tarball
```
# git diff Makefile
diff --git a/Makefile b/Makefile
index e00345e..0034d3b 100644
--- a/Makefile
+++ b/Makefile
@@ -110,6 +110,7 @@ tarball: rpmroot
        cp -r --preserve=links --parents Java/ $(RPMBUILD)/$(PKG)
        cp -r --preserve=links --parents Firefox/ $(RPMBUILD)/$(PKG)
        cp -r --preserve=links --parents Webmin/ $(RPMBUILD)/$(PKG)
+       cp -r --preserve=links --parents Chromium/ $(RPMBUILD)/$(PKG)
        cp -r JBossEAP5 $(RPMBUILD)/$(PKG)
 
        # Don't trust the developers, clean out the build
```

3) And possibly to add it to the ```*.spec``` too when it would be built on Fedora (for the start to allow users to start using it and identify issues with the OVAL checks if there are some). Later we would enable for RHEL rpm.

**Comment by redhatrises** [comment]
@iankko 

1) Good catch. Fixed
2) Okay. Fixed
3) I would like to open a separate PR for this as I would probably add Firefox and Java to the Fedora portion of the rpm.

**Comment by iankko** [comment]
@redhatrises 

Thank you for the updates. Tried on RHEL-6 with chromium-browser package and the content seems to be scanning fine. There are couple of issues in the remediation scripts (couple of rules return ```error``` to indicate the script updated the configuration, but corresponding OVAL check doesn't return PASS on re-scan / subsequent scan), but this version is sufficient for me to be merged as starting point.

ACK from me && merging.

**Comment by iankko** [comment]
@redhatrises 
> 3) I would like to open a separate PR for this as I would probably add Firefox and Java to the Fedora portion of the rpm.

Sure, go ahead (no objection from me this to be in separete PR). Thanks for adding the support!

**Pull request merged**